src/mmw/apps/modeling/tasks.py

```diff
@@ -211,23 +210,18 @@ def run_tr55(census, model_input):
     A thin Celery wrapper around our TR55 implementation.
     """
 
-    et_max = 0.207
     precip = _get_precip_value(model_input)
 
     if precip is None:
         raise Exception('No precipitation value defined')
 
-    # TODO: These next two lines are just for
-    # demonstration purposes.
     modifications = get_census_modifications(model_input)
```

Comment:
No, it should not have been.  I guess I subconsciously remembered making this change twice before (my previous Geotrellis P/Rs)